puppeteer-test/tslint.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
[Opt / no action required] Is there some way we can automatically inherit / share our tslint with the /ui/ directory? It feels like linting rules should be treated as a universal style enforcer for our wider codebase.

Reply:
for now `tsconfig.json` extends `../ui/tsconfig.json`. I don't know how to extends tslint but I'll look into it.